CMakeLists.txt

```
@@ -27,13 +27,19 @@ if(NOT CMAKE_INSTALL_PREFIX)
 endif()
 
 project(dawn C CXX)
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.14.5)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 set(YODA_ROOT CACHE PATH "path to yoda package")
 
 if("${YODA_ROOT}" STREQUAL "")
-  message(FATAL_ERROR "YODA_ROOT not found! Try specifying it in the environment via -DYODA_ROOT=<>")
+  include(FetchContent)
+  FetchContent_Declare(
+    yoda
+    GIT_REPOSITORY https://github.com/MeteoSwiss-APN/yoda.git
+  )
+  FetchContent_Populate(yoda)
```

```suggestion
  if(NOT yoda_POPULATED)
    FetchContent_Populate(yoda)
  endif()
```
This would be the safer way to do it.